controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -1307,6 +1320,77 @@ int cli_bml::get_dcs_scan_results_caller(int numOfArgs)
     return -1;
 }
 
+/**
+ * Caller function for client_get_client_list_caller.
+ *
+ * @param [in] numOfArgs Number of received arguments
+ * @return 0 on success.
+ */
+int cli_bml::client_get_client_list_caller(int numOfArgs)
+{
+    if (numOfArgs == 0) {
+        return client_get_client_list();
+    }
+    return -1;
```

Comment:
```suggestion
    if (numOfArgs != 0) {
        return -1;
    }
    
    return client_get_client_list();
```

Reply:
If you look at other `_caller` functions, they all follow this structure:
```
if (numOfArgs checks) {
     /*run command*/
}

return -1
```